.github/workflows/backend-test.yml

```diff
@@ -88,9 +88,26 @@ jobs:
       matrix:
         tarantool: ['1.10', '2.7', '2.8', '2.10', '2.11']
         etcd: ['v2.3.8']
+        metrics: ['']
         include:
           - tarantool: '2.11'
             etcd: 'v3.5.0'
+            metrics: ''
+          - tarantool: '2.10'
+            etcd: 'v2.3.8'
+            metrics: '0.9.0'
+          - tarantool: '2.10'
+            etcd: 'v2.3.8'
+            metrics: '0.13.0'
+          - tarantool: '2.10'
+            etcd: 'v2.3.8'
+            metrics: '0.14.0'
+          - tarantool: '2.10'
+            etcd: 'v2.3.8'
+            metrics: '0.15.1'
+          - tarantool: '2.10'
+            etcd: 'v2.3.8'
+            metrics: '0.16.0'
```

Comment:
I propose to leave only the latest metrics version. Tests in cartridge are hard-weighted, so several additional workflows could be bad for PR performance.

Reply:
As we won't remove the code from the metrics repo, maybe we should add it as a submodule?

Reply:
And maybe we should add metrics to the rockspec as a dependency?

Reply:
> And maybe we should add metrics to the rockspec as a dependency?

But cartridge works fine without metrics, unless you decide to enable `cartridge.roles.metrics` with no `metrics`.

Reply:
> But I have heard that building a Lua module with submodules is rather painful.

Or maybe not. I'll try this.